elastica/rod/cosserat_rod.py

```diff
@@ -156,6 +190,9 @@ def __init__(
         internal_couple,
         damping_forces,
         damping_torques,
+        ring_rod_flag,
```

Comment:
Just a thought, wouldn't it be better to just create `CosseratRing` that derives from `CosseratRod` and override necessary functions, instead of having a flag?

Reply:
I think having a flag is useful. For example block structure in backend needs to know how many ring rods are there. We can think better implementations in future, I am open to change. But for time being lets merge without making this change.